tests/integ/test_function.py

```diff
@@ -209,19 +209,31 @@ def test_order(session):
 @pytest.mark.localtest
 def test_current_date_and_time(session):
     max_delta = 1
-    df = session.create_dataframe([1]).select(
-        current_date(), current_time(), current_timestamp()
+    df = (
+        session.create_dataframe([1, 2])
```

Comment:
The main reason this bug wasn't observed initially was because constant functions returned one row previously which appears to be correct when there is only one row in the input data. By making the test have two rows I was able to observe the bug by running the test.